app/lib/handlers/base.py

```
@@ -45,15 +64,28 @@ def verify(self, bot, chat_id, user_id, user_name, callback_chat_id):
         if self.is_verified(user_id):
             return
 
-        db.session.add(
-            Human(
-                user_id=user_id,
-                user_name=user_name,
-                verified=True,
-                verification_date=datetime.today(),
+        if self.is_exists(user_id):
+            db.session.query(       
+                Human.query.filter(                                                                             
+                    Human.user_id == str(user_id)                                                                                                      
+                    ).update(                                                                                                                                         
+                        {                                
+                            "verification_date": datetime.today(),
+                            "verified": True                                                                                                                          
+                        }                                                                                                                                             
+                        ))                                                                                                                                                 
+            db.session.commit() 
+        else:
+            # Should never be called
```

Should we just throw an exception here at this point then? Why add anyway?